### DISCUSSION TITLE
"Use as webserver" 404 behavior

### DISCUSSION THREAD

**Discussion by YeapGuy** [ZoeyVid/NPMplus#717] [Q&A] [answered]
Why do all non-existent pages get 307 redirected to the index when using NPM as a webserver (as in the README)? I want proper 404 behavior from a webserver...

**Comment by Zoey2936** [comment, marked as answer]
This is because of: https://github.com/ZoeyVid/NPMplus/blob/16e85e8dcb0ca85546e61c680c0db5bc330430eb/rootfs/usr/local/nginx/conf/nginx.conf#L61
I will add an option to change this

**Reply by Zoey2936** [reply]
will be released now

**Reply by Zoey2936** [reply]
see the compose.yaml
